libs/partners/openai/langchain_openai/chat_models/base.py

```diff
@@ -1463,6 +1468,43 @@ def is_lc_serializable(cls) -> bool:
         """Return whether this model can be serialized by Langchain."""
         return True
 
+    def _should_stream_usage(
+        self, stream_usage: Optional[bool] = None, **kwargs: Any
+    ) -> bool:
+        """Determine whether to include usage metadata in streaming output.
+
+        For backwards compatibility, we check for `stream_options` passed
+        explicitly to kwargs or in the model_kwargs and override self.stream_usage.
+        """
+        stream_usage_sources = [  # order of preference
+            stream_usage,
+            kwargs.get("stream_options", {}).get("include_usage"),
+            self.model_kwargs.get("stream_options", {}).get("include_usage"),
```

Comment:
what about self.stream_usage

Reply:
it's below in the return statement. all the values in this list are `bool | None`.

Reply:
whats wrong with having a bool in the list

Reply:
Added to the list, the linter just gets confused and requires a return statement outside the loop. I felt like `self.stream_usage` is the best default return so it appears there as well.